platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/PackagePageRenderer.java

```diff
@@ -32,7 +32,7 @@ protected void renderBreadcrumbs(SimpleHtmlWriter htmlWriter) throws IOException
 
     private void renderClasses(SimpleHtmlWriter htmlWriter) throws IOException {
         htmlWriter.startElement("table");
-        htmlWriter.startElement("thread");
+        htmlWriter.startElement("thead");
```

Comment:
AFAICT, this had no impact on the rendered page.

Reply:
As we don't render a `tbody`, yes, this element effectively did nothing.